### ISSUE TITLE
Incorrect number of arguments when using DESeq with glmGamPoi

### ISSUE THREAD

**Issue by TriLe965** [mikelove/DESeq2#40]
Hi. I ran into very strange issue every time I use `DESeq()` on my scRNA-Seq data. There is information of the package version in the screenshot below. I installed them all using the latest version on github via `devtools`.

Please let me know what should I do to overcome this problem. I'm using Windows 10. My colleague ran the same script on his mac and it worked just fine...

![image](https://user-images.githubusercontent.com/11041356/119659701-016da480-be61-11eb-97ef-eeaea9b597c1.png)

**Comment by mikelove** [comment]
hi,

I checked the source code, and every place where we call fitDispGrid we are supplying the correct number of arguments.

Given that we just have a new Bioc release, could you update to R 4.1 and try the Windows binaries (1.32 for DESeq2, 1.4 for glmGamPoi)?

**Comment by TriLe965** [comment]
For some reasons, I need to stay on R 3.6.2. I notice something different when I installed the same library but with different method:
1. `devtools::install_github('mikelove/DESeq2', ref='442f6ae')`
2. Manually cloned git repository (442f6ae) and ran `R CMD INSTALL ...` 

![image](https://user-images.githubusercontent.com/11041356/119769742-99b16b00-beed-11eb-8f5c-2fd12a8ed3be.png)

This is when I installed two times with method (1):
![image](https://user-images.githubusercontent.com/11041356/119769962-15abb300-beee-11eb-8ee0-6c3a754bc0e5.png)

This is when I installed two times with method (2):
![image](https://user-images.githubusercontent.com/11041356/119770327-9d91bd00-beee-11eb-9540-1da6d42d0478.png)

Not sure what is happening but I guess the problem is from my end. Thanks for your prompt response!
